src/main/java/step3/model/Cars.java

```diff
@@ -0,0 +1,39 @@
+package step3.model;
+
+import step3.App;
+import step3.move.BasicMoveStrategy;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Cars {
+    private final List<Car> cars;
+
+    private Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars of(int carCount) {
+        return new Cars(
+            Stream.generate(() -> new Car(new BasicMoveStrategy()))
+                .limit(carCount)
+                .collect(Collectors.toList())
+        );
```

Comment:
```suggestion
        return Stream.generate(() -> new Car(new BasicMoveStrategy()))
                    .limit(carCount)
                    .collect(collectingAndThen(toList(), Cars::new));
```
이 부분도 다시 이런식으로 표현할 수 있답니다 ㅎㅎ